definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-resp-journey-roles-and-permissions-nonprod.json

```diff
@@ -54,5 +54,19 @@
     "CaseFieldID": "RespondentOrganisationPolicy",
     "UserRole": "citizen",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "03/02/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "D8PetitionerLastName",
```

Comment:
So that the caseworker-caa can see those fields (as columns) when displaying the list of "unassigned cases"

Reply:
@Dizrupto, I'd appreciate a quick demo or quick explanation on what the "caseworker-caa"is.
Nothing to block this PR, just a side note.

Reply:
Yep sure Victor that's a good point. 
John's explanation and the following confluence page finally made it clear for me: https://tools.hmcts.net/confluence/display/FR/Legally+Represented+Respondent+Journey
Basically when a petitioner solicitor assigns the case to a respondent solicitor organisation. The case is unassigned from the respondent's side. That is, there are no specific solicitors assigned to it. It is only assigned to the organisation as a whole.
The Caseworker-caa is the one who will be able to see those unassigned cases and assign them to a specific person (i.e. a specific solicitor) in their organisation.
See the recording in the following comment: https://tools.hmcts.net/jira/browse/DIV-6707?focusedCommentId=943369&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-943369